platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2349,7 +2349,7 @@
 		DA25D5BC1CCD9EDE00607828 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				INFOPLIST_FILE = framework/Settings.bundle/Root.plist;
+				INFOPLIST_FILE = "framework/Info-static.plist";
```

Comment:
This build setting is part of the `settings` target, which produces Settings.bundle, not Mapbox.framework. Doesn’t this mean Settings.bundle will contain an Info.plist that describes the entire framework?